components/ILIAS/Course/classes/Objectives/Setup/class.ilMigrateCourseCertificateProviderDBUpdateSteps.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+use ILIAS\Course\Certificate\CoursePlaceholderValues;
+
+class ilMigrateCourseCertificateProviderDBUpdateSteps implements ilDatabaseUpdateSteps
+{
+    protected ilDBInterface $db;
+
+    public function prepare(ilDBInterface $db): void
+    {
+        $this->db = $db;
+    }
+
+    public function step_1(): void
+    {
+        $this->db->update(
+            'il_cert_cron_queue',
+            ['adapter_class' => [ilDBConstants::T_TEXT, CoursePlaceholderValues::class]],
+            ['adapter_class' => [ilDBConstants::T_TEXT, 'ilCoursePlaceholderValues']]
+        );
+    }
+}
```

Comment:
This is the wrong place, since the Course should not hack into certificate tables. I guess the class names are used as IDs or something? This fact should also be hidden from the consumers. Please move this part to the certificate component.